_data/projects/learn-git.yml

```
@@ -0,0 +1,12 @@
+name: Learn Git - First Contributions
+desc: This is perhaps one of the easiest ways to get introduced to open source
+site: https://github.com/rcallaby/Learn-Git/blob/main/First-Contributions/firstcontribution.md
+tags:
+  - learngit
+  - opensource
+  - firstcontribution
+  - git
+  - github
+upforgrabs:
+  name: Learn Git - First Contributions
+  link: https://github.com/rcallaby/learn-Git/labels/up-for-grabs
```

I think this is the right values for this area:

```suggestion
  name: up for grabs
  link: https://github.com/rcallaby/Learn-Git/labels/up%20for%20grabs
```